src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -308,12 +308,12 @@ export default {
 		},
 
 		/**
-		 * When this component is used to send message to a breakout room we
-		 * pass an id of modal containing component to render properly.
+		 * Selector for popovers and pickers to be rendered inside container properly.
+		 * If undefined, subcomponents treat it as document.body.
 		 */
-		containerId: {
+		container: {
 			type: String,
-			default: null,
+			default: undefined,
 		},
```

Comment:
I'd remove comment about "`If undefined, subcomponents treat it as document.body.`".

This fact is out of scope of this component. It is some fact about implementation of a component from another library. We don't know, what value it uses by default (it may change any time), and actually it is not important to the component's user: we don't need container to be exactly `document.body` (otherwise we should set `'body'` as the default value).

At the same time more important thing is not mentioned: the container have to be `box-sizing: content-box`.